.github/workflows/lint.yml

```diff
@@ -26,7 +26,7 @@ jobs:
       - uses: golangci/golangci-lint-action@v3.2.0
         with:
           # Required: the version of golangci-lint is required and must be specified without patch version: we always use the latest patch version.
```

Comment:
[optional][nit] this comment no longer appears relevant for golangci-lint-action 3.0.0+ based on
```
# Optional: version of golangci-lint to use in form of v1.2 or v1.2.3 or `latest` to use the latest version
    version: v1.29
```
from https://github.com/golangci/golangci-lint-action 

so proposal to remove the comment because we do specify a patch version
```suggestion

```